Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed:
+                                                    step.Status = "Passed";
+                                                    step["ST_ACTUAL"] = "Passed as expected";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped:
+                                                    //step.Status = "No Run";
+                                                    step.Status = "N/A";
+                                                    step["ST_ACTUAL"] = "Skipped";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Pending:
+                                                    step.Status = "No Run";
+                                                    step["ST_ACTUAL"] = "Was not executed";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Running:
+                                                    step.Status = "Not Completed";
+                                                    step["ST_ACTUAL"] = "Not Completed";
+                                                    break;
+                                            }
+                                        }
+                                        else
+                                        {
+                                            //Step not exist in Ginger so left as "No Run" unless it is step data
+                                            if (step.Name.ToUpper() == "STEP DATA")
+                                                step.Status = "Passed";
+                                            else
+                                                step.Status = "No Run";
+                                        }
+                                        step.Post();
+                                        index++;
+                                    }
+
+                                    //get all execution status for all steps
+                                    ObservableList<string> stepsStatuses = new ObservableList<string>();
+                                    foreach (Step step in stepsList)
+                                        stepsStatuses.Add(step.Status);
+
+                                    //update the TC general status based on the activities status collection.                                
+                                    if (stepsStatuses.Where(x => x == "Failed").Count() > 0)
+                                        run.Status = "Failed";
+                                    else if (stepsStatuses.Where(x => x == "No Run").Count() == stepsList.Count || stepsStatuses.Where(x => x == "N/A").Count()== stepsList.Count)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)